### PR TITLE
release omake-0.10.6

### PR DESCRIPTION
**Pull request by gerdstolpmann**
new upstream release. omake is now compatible with OCaml-5.

### CODE REVIEW

File: packages/omake/omake.0.10.6/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+build: [
+  ["./configure" "-prefix" "%{prefix}%"]
+  [make]
+]
+
+install: [
+  [make "install"]
+]
+
+remove: [
```
Comment (mseri) [inline packages/omake/omake.0.10.6/opam:23]: This can be dropped now

Comment (mseri) [reply packages/omake/omake.0.10.6/opam]: (done in separate commit)

**Comment by mseri** [comment]
Can you please check if the failres are general incompatibility with ocaml 5 (they involve bigarray of unix.a after all) or if they are related to omake?

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**